jenkins/ci.opensuse.org/openstack-trackupstream.yaml

```diff
@@ -105,6 +105,7 @@
             "openstack-trove",
             "python-monasca-common",
             "python-monasca-statsd",
+            "python-networking-cisco",
```

Comment:
It's red across the board, let's just remove it from line 86 above completely?

Reply:
> It's red across the board, let's just remove it from line 86 above completely?

That'll work too. I was following the previous pattern of skipping the vmware packages. But if the package is no longer needed, we can remove it altogether.

Reply:
As of now https://opendev.org/x/networking-cisco/commits/branch/master didn't have any commit since the 7.0.0 release we ship, except the Opendev migration. https://pypi.org/project/networking-cisco/ is the same. The package is not present in rpm-packaging. The _service file doesn't contain anything that will make it automatically update. The job is now green, but it will AFAIK never do anything.

Yes, removal from this job makes more sense.